src/viz/viz.jl

```diff
@@ -29,5 +29,6 @@ function outcome_map!() end
 # Clustering
 function ts_cluster() end
 function ts_cluster!() end
+function spatial_clustering() end
```

Comment:
Change `spatial_clustering` -> `spatial_cluster` for consistency with `ts_cluster` ?

Reply:
I have changed it to ts_spatial_clustering because it is in the same context, what do you think?